packages/react/src/components/TreeView/TreeNode.tsx

```diff
@@ -92,6 +102,9 @@ export type TreeNodeProps = {
   onToggle?: UncontrolledOnToggle | ControlledOnToggle;
   /**
    * Callback function for when any node in the tree is selected
+   *
+   *     @deprecated The `onTreeSelect` prop for `TreeNode` has
+   *     been deprecated after the introduction of context. It will be removed in the next major release.
```

Comment:
```suggestion
   * @deprecated The `onTreeSelect` prop for `TreeNode` has
   * been deprecated after the introduction of context. It will be removed in the next major release.
```